packages/paste-core/components/table/src/types.ts

```diff
@@ -0,0 +1,74 @@
+import * as PropTypes from 'prop-types';
+import {WidthOptions, isWidthTokenProp} from '@twilio-paste/style-props';
+
+export type AlignmentOptions = 'left' | 'center' | 'right';
+export type TableVariants = 'auto' | 'fixed';
+export type VerticalAlignmentOptions = 'top' | 'middle' | 'bottom';
```

Comment:
You could potentially keep these types in sync with the prop-types below by using constants.

Then you can derive the types from the constant, and populate the oneOf values from the constant too. That way it's a single update in the future to keep them in sync with each other